packages-common/pluggable-widgets-generator/generators/app/lib/utils.js

```diff
@@ -2,74 +2,37 @@ const { join } = require("path");
 const { access, readdir } = require("fs").promises;
 
 function getWidgetDetails(answers) {
-    const widget = Object.create(answers);
-
-    Object.defineProperties(widget, {
-        name: {
-            get() {
-                return answers.name.replace(/(^|\s)\S/g, l => l.toUpperCase()); // Capitalize first letter if it's not
-            }
-        },
-        packageName: {
-            get() {
-                return answers.name.toLowerCase();
-            }
-        },
-        packagePath: {
-            get() {
-                return answers.organization.trim().toLowerCase();
-            }
+    return {
+        ...answers,
+        name: answers.name.replace(/(^|\s)\S/g, l => l.toUpperCase()), // Capitalize first letter if it's not
+        packageName: answers.name.toLowerCase(),
+        packagePath: answers.organization.trim().toLowerCase(),
+        projectPath: answers.projectPath.replace(/\\/g, "\\\\"),
+        get isPlatformWeb() {
```

Comment:
You can also make all of these to a simple field, i guess...

Reply:
Hmm, isn't nice to store derivable data

Reply:
we don't do it for `projectPath`

Reply:
Changed it as discussed